src/Server.js

```diff
@@ -71,10 +75,28 @@ function middleware(bundler) {
       }
     }
 
-    function send500() {
-      res.setHeader('Content-Type', 'text/plain; charset=utf-8');
+    function send500(error) {
+      res.setHeader('Content-Type', 'text/html; charset=utf-8');
       res.writeHead(500);
-      res.end('🚨 Build error, check the console for details.');
+      let errorMesssge = '<h1>🚨 Build Error</h1>';
+      if (process.env.NODE_ENV === 'production') {
+        errorMesssge += '<p><b>Check the console for details.</b></p>';
```

Comment:
Not sure if this is necessary, the built-in parcel server should never be used in production

Reply:
Parcel can be used as middleware in a production server, so I guess it's good to check this though.

Reply:
Yeah it should never happen but just in case...